assets/src/scss/components/_buttons.scss

```diff
@@ -176,7 +176,7 @@ $donate-button-box-shadow: 0 2px 5px rgba(0, 0, 0, .25);
 .btn-donate {
   --button-donate-- {
     background: $orange;
-    color: $grey-80;
+    color: $white;
```

Comment:
I think this applies to as my last comment.

Reply:
These work as expected so I think it's fine!

Reply:
Oh yea, my bad